packages/core/src/App/Containers/RealAccountSignup/address-details-form.js

```diff
@@ -14,12 +14,25 @@ const address_details_config = ({ account_settings }) => {
             rules: [
                 ['req', localize('Address line 1 is required')],
                 ['address', localize('Address is not in a proper format')],
+                [
+                    'length',
+                    localize('You should enter a value between {{min}} and {{max}} characters.', { min: 1, max: 70 }),
+                    { min: 1, max: 70 },
+                ],
+                ['po_box', localize('P.O. Box is not accepted in address')],
             ],
         },
         address_line_2: {
             supported_in: ['svg', 'iom', 'malta', 'maltainvest'],
             default_value: account_settings.address_line_2 ?? '',
-            rules: [['length', localize('Address line 2 is not in a proper format'), { min: 0, max: 30 }]],
+            rules: [
+                [
+                    'length',
+                    localize('You should enter a value between {{min}} and {{max}} characters.', { min: 0, max: 70 }),
+                    { min: 0, max: 70 },
+                ],
+                ['po_box', localize('P.O. Box is not accepted in address')],
```

Comment:
do we need to pass this again? `localize('P.O. Box is not accepted in address')`

Reply:
Yep, it still expects a `message` to be passed, similar to `length`, `address` and `req` rules above.